activity/scraper.py

```diff
@@ -0,0 +1,135 @@
+import requests
+import json
+import datetime
+import os
+import calendar
+from dateutil import parser, relativedelta
+
+class Scraper():
+    '''
+    This is the class responsible for scraping provided issues into a
+    dictionary containing just statistical information of the data.
+    '''
+
+    # Count of months/weeks/days respectively to be scraped in past.
+    month_count = 12
```

Comment:
I am 99% sure these constants can be obtained from `calendar`.  If not, they should be CONSTANTS at the top of the module.

Reply:
hmm, I chose these constants randomly myself. They're not representative of anything, just a number I think is good for display.(so no calendar use here)